lib/internal/bridge/characteristics_mixin.dart

```diff
@@ -153,7 +153,7 @@ mixin CharacteristicsMixin on FlutterBLE {
         .receiveBroadcastStream()
         .map(
           (rawJsonValue) =>
-              _parseCharacteristicResponse(peripheral, rawJsonValue),
+              _parseCharacteristicResponseWithValue(peripheral, rawJsonValue),
```

Comment:
`_parseCharacteristicWithValueResponse`? CharacteristicWithValue is a Dart type we've created.

Reply:
Renamed to _parseCharacteristicWithValueResponse